packages/gestalt/src/Table.js

```diff
@@ -0,0 +1,81 @@
+// @flow
+import * as React from 'react';
+import styles from './Table.css';
+
+const TableBody = ({ children }: { children: React.Node }) => (
+  <tbody>{children}</tbody>
+);
+
+const TableCell = ({
+  children,
+  colSpan,
+  rowSpan,
+}: {
+  children: React.Node,
+  colSpan?: number,
+  rowSpan?: number,
+}) => {
+  let props = { className: styles.td };
+  if (colSpan) {
+    props = { ...props, colSpan };
+  }
+  if (rowSpan) {
+    props = { ...props, rowSpan };
+  }
+
+  return <td {...props}>{children}</td>;
```

Comment:
@PayalModi instead, could we write it as the following:

```
return <td className={styles.td} colSpan={colSpan} rowSpan={rowSpan} />
```

We try to avoid prop spreading as much as possible to make the code more explicit and easier to reason about. When `colSpan` is `null` or `undefined`, the prop should not get rendered in the DOM.

![image](https://user-images.githubusercontent.com/127199/81009914-52e86300-8e0a-11ea-9d09-2f4d1664d1ad.png)

Reply:
This is much easier to read, thanks for the suggestion!